introduction.md

```diff
@@ -1 +1,6 @@
-Hi it's darrenthill. My favorite Code School path is JavaScript
\ No newline at end of file
+* This is an intro.
+* My favorite Code School path is Electives.
+* On my way to becoming a GitHub Master. I love Ruby and Javascript.
+* Ruby on Rails
+* Git
+* Killing history using git rebase
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?